prow/tide/tide.go

```diff
@@ -1089,13 +1090,12 @@ func (c *Controller) mergePRs(sp subpool, prs []PullRequest) error {
 		}
 
 		commitTemplates := tideConfig.MergeCommitTemplate(config.OrgRepo{Org: sp.org, Repo: sp.repo})
-		keepTrying, err := tryMerge(func() error {
+		keepTrying, hasMerged := c.tryMerge(func() error {
 			ghMergeDetails := c.prepareMergeDetails(commitTemplates, pr, mergeMethod)
 			return c.ghc.Merge(sp.org, sp.repo, int(pr.Number), ghMergeDetails)
 		})
-		if err != nil {
-			log.WithError(err).Error("Merge failed.")
-			errs = append(errs, err)
+		if !hasMerged {
+			errs = append(errs, errors.New("Failed merging"))
```

Comment:
Why not simply log it on Debug level here? The errors in `tryMerge` are completely out of context and don't include the info that whatever happened happend when trying to merge

Reply:
Good idea, updated